backup.sh

```diff
@@ -72,7 +72,7 @@ function sqlcmd_with_backoff {
 }
 
 SQLCMD=/opt/mssql-tools/bin/sqlcmd-13.0.1.0
-DB_INSTANCE_IDENTIFIER=$DB_ENGINE-$SERVICE_NAME-auto-restore
+DB_INSTANCE_IDENTIFIER=$DB_ENGINE-$SERVICE_NAME-$DEPLOY_ENV-auto-restore
```

Comment:
I don't think the $DEPLOY_ENV is needed here since the DB Instance will get created in the environment itself - the names shouldn't collide with each other between prod and stage here